src/dpq2/conv/from_d_types.d

```diff
@@ -34,6 +34,9 @@ if(isNumeric!(T))
     static if (is(T == enum))
     {
         alias OType = OriginalType!T;
+
+        static assert(is(OType == int));
```

Comment:
Why? As mentioned postgresql has many int types, why reduce it here just to int based enums?